tests/snapd

```diff
@@ -99,12 +90,19 @@ done
 if [ "${ID}" != "ubuntu-core" ]; then
     (
         set -e
-        usermod -G lxd -a jenkins || true
+
+        test_user="testuser"
+        useradd -m ${test_user} || true
+        usermod -G lxd -a ${test_user}
         LXC=$(which lxc)
-        su jenkins -c "${LXC} info"
+        su ${test_user} -c "${LXC} info"
+        userdel -r ${test_user} 2>/dev/null || true
     )
 fi
 
+# Check that preseed via snap works
+lxc config show | grep -qF "core.https_address: '[::]:8443'" 
+
 # Container creation test
 lxc launch images:alpine/edge unprivileged
 lxc launch images:alpine/edge privileged -c security.privileged=true
```

Comment:
Let's use `ubuntu-minimal-daily:24.04` here please.

Reply:
Updated all uses of `alpine/edge` to `ubuntu-minimal-daily:24.04` since it seems like this is what is used in other tests.